virttest/utils_test/__init__.py

```diff
@@ -2069,7 +2069,13 @@ def __init__(self, vm, stress_type, params, download_url="", make_cmds="",
         :param stress_args: the arguments of the stress tool
         :param download_type: currently support "git" or "tarball"
         """
-        super(VMStress, self).__init__(stress_type, params, download_url=download_url,
+        # This enables VM specific stress params like stress_cmds_virt-tests-vm1, etc.,
+        # to run different stress type on different VM
+        self.stress_type = params.get("stress_type_%s" % vm.name, stress_type)
```

Comment:
@balamuruhans `stress_type` need not be appended with vm.name always, if it is given in that way fine, otherwise also it should work, right now it mandates it

Reply:
@sathnaga  it doesn't mandates currently, it just checks whether user wants to configure separate `stress_type_VM` for any specific VM, else it falls back to `stress_type` argument.

Reply:
@balamuruhans sure, I was carried away with `load_stress(stress_type,` ` stress_type in ['stress_in_vms', 'iozone_in_vms']:` implementation and thought there was an `stress_type` in first place, looks like not, got it now,

LGTM.